numpy/core/defchararray.py

```diff
@@ -1817,7 +1839,27 @@ def isdecimal(a):
     --------
     unicode.isdecimal
 
-    """
+    Examples
```

Comment:
Same comment as the other docstring - let's condense this down to array inputs only.

Reply:
condensed to array input examples